activesupport/lib/active_support/core_ext/module/attribute_accessors_per_thread.rb

```diff
@@ -65,7 +65,7 @@ def #{sym}
   #   Current.user = "DHH"
   #   Thread.current[:attr_Current_user] # => "DHH"
   #
-  # If you want to opt out the instance writer method, pass
+  # If you want to opt out the creation of the instance writer method, pass
```

Comment:
I think it sounds better with this extra "of": "If you want to out out of the creation of the instance writer".